packages/teleterm/src/mainProcess/runtimeSettings.ts

```diff
@@ -209,4 +211,32 @@ function getUnixSocketNetworkAddress(socketName: string) {
   return `unix://${path.resolve(app.getPath('userData'), socketName)}`;
 }
 
+const UUID_V4_REGEX =
+  /^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i;
+
+function loadInstallationId(): string {
+  const filePath = path.resolve(app.getPath('userData'), 'installation_id');
+  const writeNewId = (): string => {
+    try {
```

Comment:
Wouldn't it be better to wrap the `try` as tightly as possible here, that is only around `fs.writeFileSync`?

Reply:
Yeah, done.